src/utils/components/FeedbackPopup.vue

```diff
@@ -0,0 +1,74 @@
+<script setup>
+import { computed } from 'vue'
+import { useI18n } from 'vue-i18n'
+import { useStore } from 'vuex'
+
+import ErrorWindow from '@/utils/components/ErrorWindow.vue'
+import WarningWindow from '@/utils/components/WarningWindow.vue'
+const store = useStore()
+const i18n = useI18n()
+const error = computed(() => {
+    if (store.state.ui.errors.size > 0) {
+        return store.state.ui.errors.values().next().value
+    }
+    return null
+})
+const warning = computed(() => {
+    if (store.state.ui.warnings.size > 0) {
+        return store.state.ui.warnings.values().next().value
+    }
+    return null
+})
+</script>
+
+<template>
+    <div class="feedback-window">
+        <ErrorWindow
```

Comment:
might be nice to have some kind of an error/warning count if more than one

Reply:
I think @ismailsunni had something in one of his PR. If this is not on its way to be merged, I'll add this too.

Reply:
Joining the discussion, in my PR, I kept adding the error to the store even if they are the same. But I show the number of same error in the component title. When user close the error, I remove the same error in the store.

So, it depends on the error count. Is it the number of ALL error or the number of the SAME error?

Reply:
btw, I just had this corner case in my mind. Let's say, there is currently error message A. The user doesn't close it. Then a new error message B pops up (covering the error message A). The user doesn't close it also. When there is a new error message A, it won't appear above the error message B (because it's not added in the store). Do you think it's ok to not show the latest error message? It might give a wrong feedback to the user.

Again, this is a corner case.

Reply:
The current behavior of the messages is to pick the first of the list, not the last. The corner case would be :
User gets message A, doesn't close it
User gets message B, doesn't see it (but a little '2' will be shown)
User gets message A again, no change visible.

We are working under the assumption that people want to close all the red things everywhere ^^. If they decide to ignore error messages, it's their own responsibility ^^